context.go

```diff
@@ -170,8 +170,16 @@ type cbContext struct {
 	wg     *sync.WaitGroup
 }
 
+func WithCtxEmitHeaders(headers map[string][]byte) ContextOption{
```

Comment:
A small description would be nice :)

Reply:
Oops, I've added a comment. I also noticed this should probably be in the options file and moved it.